src/Framework/TestSuite.php

```diff
@@ -796,7 +796,7 @@ public function warnings(): array
     }
 
     /**
-     * @return array<ExecutionOrderDependency>
+     * @return list<ExecutionOrderDependency>
```

Comment:
The properties in the TestSuite can be null or `list<ExecutionOrderDependency>`. Does that mean that this may also return a null value? (unless you fixed my previous question)

Reply:
No, the `null` is never returned via the public interface of `TestSuite`. The nullability is to signal "the dependency tree has not been calculated yet" and hidden behind the friendly face of `provides()/requires()`